estuary/utils/story.py

```diff
@@ -0,0 +1,468 @@
+
+        query = ('MATCH (next_node:{next_label})-[:{rel}]-(sibling:{curr_label})'
+                 'WHERE id(next_node)= {next_node_id}').format(
+            next_label=story_node.__label__, rel=relationship, curr_label=siblings_node_label,
+            next_node_id=story_node.id)
+
+        if count:
+            query += ' RETURN COUNT(sibling) as count'
+            results, _ = db.cypher_query(query)
+            count = results[0][0]
+            if count == 0:
+                return count
+            # We reduce the count by one to ignore the node already being shown in the story
+            return count - 1
+        else:
+            query += ' RETURN sibling'
+            results, _ = db.cypher_query(query)
+            return results
+
+    def format_story_results(self, results, requested_item):
+        """
+        Format story results from Neo4j to the API format.
+
+        :param list results: nodes in a story/path
+        :param EstuaryStructuredNode requested_item: item requested by the user
+        :return: results in API format
+        :rtype: dict
+        """
+        data = []
+        for i, node in enumerate(results):
+            if node.id == requested_item.id:
+                requested_node_index = i
+                serialized_node = node.serialized_all
+            else:
+                serialized_node = node.serialized
+            serialized_node['resource_type'] = node.__label__
+            serialized_node['display_name'] = node.display_name
+            data.append(serialized_node)
+        return {
+            'data': data,
+            'meta': {
+                'story_related_nodes_forward': list(self.get_sibling_nodes_count(results)),
+                'story_related_nodes_backward': list(
+                    self.get_sibling_nodes_count(results, reverse=True)),
+                'requested_node_index': requested_node_index
+            }
+        }
+
+    def set_story_labels(self, requested_node_label, results, reverse=False):
+        """
+        Replace Neo4j labels with appropriate labels of the story flow.
+
+        :param string requested_node_label: label of the node requested by the user
+        :param list results: nodes in a story/path
+        :kwarg bool reverse: determines if the results are in reverse order of the story flow
+        :return: results with story/path labels
+        :rtype: list
+        """
+        if not results:
+            return results
+
+        idx = 0
+        if reverse:
+            idx = len(results) - 1
+        delta = -1 if reverse else 1
+        node_idx = self.story_flow_list.index(requested_node_label)
+        while (0 <= idx < len(results)):
+            results[idx].__label__ = self.story_flow_list[node_idx]
+            idx += delta
+            node_idx += delta
+        return results
+
+    def get_siblings_description(self, story_node_display_name, story_node_story_flow, backward):
```

Comment:
This should be a static method since you are passing in all the information and not using any information from the class.